python/tvm/contrib/cutlass/build.py

```diff
@@ -105,9 +131,19 @@ def tune_cutlass_kernels(mod, sm, profile_all=True, use_multiprocessing=False, t
             MM = arg0_shape[0]
             KK = arg0_shape[1]
             NN = arg1_shape[0]
-            out = cutlass_profiler.profile(
-                MM, NN, KK, annotator.signature["ret_dtype"], profile_all, use_multiprocessing
-            )
+            out_dtype = annotator.signature["ret_dtype"]
+            if any(isinstance(s, tvm.tir.Any) for s in [MM, KK, NN]):
+                out = cutlass_profiler.get_default(out_dtype)
+                print("Picked the default kernel " + out["name"])
```

Comment:
Proper logging.

Reply:
Replaced with `logging.info` but turning on the logging in the test script produces log from other modules:

```
$ python test_cutlass.py 
INFO:cutlass:Picked the default kernel cutlass_tensorop_h16816gemm_128x256_32x3_tn_align4
INFO:te_compiler:Using injective.cpu for add based on highest priority (10)
INFO:te_compiler:Using injective.cpu for divide based on highest priority (10)
INFO:te_compiler:Using reduce.cpu for prod based on highest priority (10)
INFO:te_compiler:Using injective.cpu for multiply based on highest priority (10)
INFO:te_compiler:Using injective.cpu for cast based on highest priority (10)
INFO:te_compiler:Using injective.cpu for add based on highest priority (10)
INFO:te_compiler:Using injective.cpu for divide based on highest priority (10)
INFO:te_compiler:Using reduce.cpu for prod based on highest priority (10)
INFO:te_compiler:Using injective.cpu for multiply based on highest priority (10)
INFO:te_compiler:Using dense_cublas.cuda for nn.dense based on highest priority (25)
```

Do you know how to log only `cutlass` ones?

Reply:
This is because you set the logging level to INFO in the test script. It's not trivial, unfortunately, in Python to filter logging in the way you want. Here we could have two options:

1. Keep the current level so that users have to explicitly set the logging level to display it.
2. Use WARNING level.

Reply:
ok if this is not simple, I'll defer it for now.